### ISSUE TITLE
Kami Lua retrieving variable in configuration file 

### ISSUE THREAD

**Issue by jpskriver** [kamailio/kamailio#2822]
### Description
Suppose that there is an variable define in `kamailio-basic-kemi.cfg` with:
```
#!define MYVAR 1.2.3.4
```

Then in the `kamailio-basic-kemi-lua.lua`

- How can we get that variable?

### Possible Solutions
Is there any alternative solution like set global via app_lua parameter, example:
```
modparam("app_lua", "load", "/usr/local/etc/kamailio/lua/myscript.lua")
modparam("app_lua", "varset", "pstngw=s:sip:10.10.10.10")
```

**Comment by arsperger** [comment]
for KEMI there is an option which was recently merged to the master
you can get value of #!defined by using KSR.kx.get_def("VAR") 

#!define MYVAR 1234 

then in your script you can use
KSR.kx.get_def("MYVAR")

there was a mail thread on this topic in users list

**Comment by henningw** [comment]
As mentioned from @arsperger, should be already implemented with 851a8c0f0
